api/edge_api/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```

Comment:
Don't we already have an app named `edge_api`? :confused:

Reply:
Nope — we have `"edge_api.identities"` which, amusingly, cannot be added to `INSTALLED_APPS` because it collides with `"environments.identities"`.